portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,47 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// function addRandomGreeting() {
```

Comment:
Left over code?

Reply:
There was no need for this function so I commented it initially. I forgot to delete it while pushing my code. I will delete this in next pull request.